src/util/util_errors.h

```diff
@@ -195,6 +195,10 @@ enum sssd_errors {
 #define ETIME 9935
 #endif
 
+#if !defined(ENONET) && defined(EHOSTDOWN)
+#define ENONET EHOSTDOWN
```

Comment:
Does FreeBSD have `ENETUNREACH`?

`ENONET` is used once - to translate libdbus error code:
```
{ DBUS_ERROR_NO_NETWORK,        ENONET},
```
https://dbus.freedesktop.org/doc/api/html/group__DBusProtocol.html says
"No network access (probably ENETUNREACH on a socket). "

This is an internal API and code currently doesn't have any special handling of this error code.
It's only printed in logs.

Reply:
I mean, we can change this translation.

Reply:
Sorry, I didn't mean to use `ENETUNREACH` instead of `EHOSTDOWN` to re-define `ENONET`.

I meant to use `ENETUNREACH` directly in 
https://github.com/SSSD/sssd/blob/e50533d6610776e876afb87feb74fd799f7cb18b/src/sbus/sbus_errors.c#L56
it fits better anyway and shouldn't cause any troubles.